examples/config files - basic/3 connector-ldap.yml

```diff
@@ -67,6 +70,7 @@ all_users_filter: "(&(objectClass=user)(objectCategory=person)(!(userAccountCont
 # You will likely want to replace it with a simpler query customized for your directory,
 # such as this one for Active Directory: "(&(objectCategory=group)(cn={group}))"
 # or this one for OpenLDAP: "(&(|(objectClass=groupOfNames)(objectClass=posixGroup))(cn={group}))"
+# If enabled, will be used whenever "--users group" or "--users mapped" arguments are used
```

Comment:
This is very misleading.  Remove this lie completely.  This predicate is used whenever we need the contents of a group, whether specified in the command line or specified in the group mapping.